### PR TITLE
Add outputs to tanium

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/7517

### CODE REVIEW

**Comment by Spivaka** [comment]
@liorkol - What is the war room related? this is the context.
the available time is not set as the path you gave is wrong.

**Comment by Spivaka** [comment]
@aviadl please RMD

File: Scripts/script-CommonServer.yml [hunk 1]
```diff
@@ -23,6 +23,31 @@ script: |-
...
+            var cleanContents = {};
+            var keys = Object.keys(contents);
+            for (var i = 0; i < keys.length; i++) {
+                if (contents[keys[i]] || contents[keys[i]] === false) {
+                    cleanContents[keys[i]] = contents[keys[i]];
+                }
+            }
+            return cleanContents;
+        }
+        if (obj instanceof Array) {
```
Comment (aviadl) [inline Scripts/script-CommonServer.yml]: what if you have an array of objects inside the root.
You are not going recursively here

File: Scripts/script-TaniumApprovePendingActions.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 commonfields:
   id: TaniumApprovePendingActions
-  version: -1
+  version: 4
```
Comment (aviadl) [inline Scripts/script-TaniumApprovePendingActions.yml]: version: -1

File: Scripts/script-TaniumApprovePendingActions.yml [hunk 3]
```diff
@@ -119,14 +137,20 @@ tags:
 comment: Send a request for a formatted result of a saved question. To receive the
   most up to date data, run the same command twice. See https://kb.tanium.com/SOAP
   for more information
-system: true
+enabled: true
```
Comment (aviadl) [inline Scripts/script-TaniumApprovePendingActions.yml]: what is enabled ?

File: Scripts/script-TaniumApprovePendingActions.yml [hunk 4]
```diff
@@ -119,14 +137,20 @@ tags:
...
+- contextPath: Tanium.Action.State
+  description: Action state
+- contextPath: Tanium.Action.Comment
+  description: Tanium comment
+- contextPath: Tanium.Action.PackageName
+  description: Tanium package name
+- contextPath: Tanium.Action.PackageID
+  description: Tanium package id
 scripttarget: 0
-dependson:
```
Comment (aviadl) [inline Scripts/script-TaniumApprovePendingActions.yml]: Why did you remove the dependsOn ?

File: Scripts/script-TaniumDeployAction.yml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 commonfields:
   id: TaniumDeployAction
-  version: 1
+  version: 8
```
Comment (aviadl) [inline Scripts/script-TaniumDeployAction.yml]: version : -1

File: Scripts/script-TaniumDeployAction.yml [hunk 6]
```diff
@@ -71,8 +89,15 @@ args:
...
+- contextPath: Tanium.Action.Command
+  description: Action command
+- contextPath: Tanium.Action.State
+  description: Action state
+- contextPath: Tanium.Action.Comment
+  description: Tanium comment
+- contextPath: Tanium.Action.PackageName
+  description: Tanium package name
 scripttarget: 0
-dependson:
```
Comment (aviadl) [inline Scripts/script-TaniumDeployAction.yml]: should not be removed

File: Scripts/script-TaniumShowPendingActions.yml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 commonfields:
   id: TaniumShowPendingActions
-  version: -1
+  version: 10
```
Comment (aviadl) [inline Scripts/script-TaniumShowPendingActions.yml]: -1

File: Scripts/script-TaniumShowPendingActions.yml [hunk 8]
```diff
@@ -111,13 +126,22 @@ tags:
...
+- contextPath: Tanium.Action.Comment
+  description: Tanium comment
+- contextPath: Tanium.Action.PackageName
+  description: Tanium package name
+- contextPath: Tanium.Action.PackageID
+  description: Tanium package id
+- contextPath: Tanium.Action.AvailableTime
+  description: Tanium available time
 scripttarget: 0
-dependson:
```
Comment (aviadl) [inline Scripts/script-TaniumShowPendingActions.yml]: should stay
